<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-8J5S5LH1" failures="2" tests="2" name="Test1" time="21.05" errors="0" timestamp="16 Nov 2021 12:15:28 GMT">
  <testcase classname="FWBPrac.validateTitle" name="validateHeader" time="0.278">
    <failure type="java.lang.AssertionError" message="expected [An Academy to learn Everything about Testing] but found [AN ACADEMY TO LEARN EVERYTHING ABOUT TESTING]">
      <![CDATA[java.lang.AssertionError: expected [An Academy to learn Everything about Testing] but found [AN ACADEMY TO LEARN EVERYTHING ABOUT TESTING]
at FWBPrac.validateTitle.validateHeader(validateTitle.java:48)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- validateHeader -->
  <testcase classname="FWBPrac.validateTitle" name="validationOfTitle" time="0.062">
    <failure type="java.lang.AssertionError" message="expected [FEATURED C123OURSES] but found [FEATURED COURSES]">
      <![CDATA[java.lang.AssertionError: expected [FEATURED C123OURSES] but found [FEATURED COURSES]
at FWBPrac.validateTitle.validationOfTitle(validateTitle.java:39)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- validationOfTitle -->
</testsuite> <!-- Test1 -->
